docs/ManualDeploymentGuideScalarDLAuditorOnAWS.md

```diff
@@ -59,28 +59,18 @@ so you need to add peering for internal communication between the Auditor, Ledge
 
 ### Steps
 
-* Create 3 peering connections between the Ledger, Auditor and Client VPCs based on the [AWS official guide](https://docs.aws.amazon.com/vpc/latest/peering/create-vpc-peering-connection.html).
+* Create three peering connections between the Ledger, Auditor and Client VPCs based on the [AWS official guide](https://docs.aws.amazon.com/vpc/latest/peering/create-vpc-peering-connection.html).
     * Peering between Ledger and Auditor.
     * Peering between Ledger and Client.
     * Peering between Auditor and Client.
 * Update route tables for VPC peering connection based on the [AWS official guide](https://docs.aws.amazon.com/vpc/latest/peering/vpc-peering-routing.html).
-* Add new Inbound and Outbound rule to restrict unwanted access to Ledger and Auditor.
-    * Add new Inbound rules to the Ledger Network ACLs to allow the Ledger Envoy LoadBalancer (e.g., 50051 and 50052 by default) access from the Auditor and Client.
+* Add new Inbound and Outbound rules to Ledger Network ACLs and Auditor Network ACLs to restrict unwanted access to the Ledger and Auditor.
+    * Add a new Inbound rule to allow all access from the same VPC.
         * You must set high priority to this rule.
-    * Add new Inbound rules to the Ledger Network ACLs to allow ssh access from the internet.
-        * You must set second priority to this rule.
-    * Add new Inbound rules to Ledger Network ACLs to allow all access to Ledger VPC.
-        * You must set third priority to this rule.
-    * Add new Outbound rules to the Ledger Network ACLs to allow all access to the internet.
-    * Add Ledger private subnets to subnet associations of the Ledger Network ACLs.
-    * Add new Inbound rules to the Auditor Network ACLs to allow the Auditor Envoy LoadBalancer (e.g., 40051, 40052 by default) access from the Ledger and Client.
-        * You must set high priority to this rule.
-    * Add new Inbound rules to the Auditor Network ACLs to allow ssh access from the internet.
-        * You must set second priority to this rule.
-    * Add new Inbound rules to Auditor Network ACLs to allow all access to Auditor VPC.
-        * You must set third priority to this rule.
-    * Add new Outbound rules to the Auditor Network ACLs to allow all access to the internet.
-    * Add Auditor private subnets to subnet associations of the Auditor Network ACLs.
+    * Add a new Inbound rule to allow ephemeral ports 1024–65535 access from the internet.
+        * You must set the next priority to this rule.
+    * Add a new Outbound rule to allow all access to the internet.
+    * Add private subnets of Ledger and Auditor to the subnet associations of the respective Network ACLs.
```

Comment:
1. By default, the new NACL rule will restrict all networks so we need to allow access for communication.
2. We need to add a new inbound rule to allow all access to the same VPC, otherwise, we can't use `kubectl` and `helm` command from the bastion.
3. We need to add a new inbound rule to allow ephemeral ports 1024–65535 access for the internet (we need to download Ledger, Auditor, schema-loader, Prometheus and Grafana container image from the internet).
`Since NAT Gateway uses ports 1024-65535 so we need to add a new inbound rule to allow ephemeral ports (1024–65535) for the Internet. (For example, if we try to download a container image from the Github package, the destination port would be permanent and the source port would be a random port from the ephemeral port range of 1024 - 65535.)`
* This rule allows Ledger and Auditor Envoy LoadBalancer connection between Ledger, Auditor, and Client
4. We need to add an Outbound rule to allow all access to the internet (we need to download Ledger, Auditor, schema-loader, Prometheus and Grafana container image download from the internet).

Reply:
Questions.

> We need to add a new inbound rule to allow ephemeral ports 1024–65535 access for the internet (we need to download Ledger, Auditor, schema-loader, Prometheus and Grafana container image from the internet).

* If we don't download the containers from the internet, then we don't need to add the rule?
I think, in the near future, we are adding the containers to AWS/Azure container services. If it is downloading from those services, can we restrict the source?
* Opening the wide range is OK for security? I think it's very insecure, but is this what AWS recommends or what other people do?